projects/capybara/classification/settings/htm_network.py

```diff
@@ -26,9 +26,10 @@
 INPUT_DIR = os.path.join(parentDir, 'data')
 OUTPUT_DIR = os.path.join(parentDir, 'results')
 FILE_NAMES = [
-  os.path.join('uci','body_acc_x.csv'),
-  # os.path.join('artificial','binary_ampl=10.0_mean=0.0_noise=0.0.csv'),
-  # os.path.join('artificial','binary_ampl=10.0_mean=0.0_noise=1.0.csv'),
+  os.path.join('uci','body_acc_x_inertial_signals_train.csv'),
+  os.path.join('uci','body_acc_x_inertial_signals_test.csv'),
+  #os.path.join('artificial','binary_ampl=10.0_mean=0.0_noise=0.0.csv'),
+  #os.path.join('artificial','binary_ampl=10.0_mean=0.0_noise=1.0.csv'),
```

Comment:
Updated. Note that I am not using the network API anymore. This is part of a performance optimization done after profiling `htm_network.py`.